src/mudlet.cpp

```diff
@@ -1431,6 +1431,29 @@ void mudlet::closeHost(const QString& name)
     int hostCount = mHostManager.getHostCount();
     emit signal_hostDestroyed(pH, --hostCount);
     mHostManager.deleteHost(pH->getName());
+    updateMultiviewControlsEnabled();
+}
+
+void mudlet::updateMultiviewControlsEnabled()
+{
+    const bool isEnabled = (mHostManager.getHostCount() - 1);
+    if (mpActionMultiView->isEnabled() != isEnabled){
+        mpActionMultiView->setEnabled(isEnabled);
+    }
+    if (dactionMultiView->isEnabled() != isEnabled) {
+        dactionMultiView->setEnabled(isEnabled);
+    }
+}
+
+void mudlet::reshowAllMainConsolesIfRequired()
```

Comment:
```suggestion
void mudlet::reshowAllMainConsoles()
```

Reply:
:-1: Again, the shorter wording implies something that is not the case.

Reply:
I think most people will probably understand that it only does it if required, though? Unless there's another, very similar worded method name I don't know that we gain much by adding the "IfRequired" into the name?

Reply:
The thing is, that the method makes an assessment as to whether it needs to "reshow" the main windows, but if it's assessment is wrong then it won't - the shorter name would suggest that it will always "reshow" them and I think that could be misleading.

Reply:
The IfRequired is too imprecise for me. Maybe IfInMultiview? Again, I don't think this name is too long.

Reply:
We can make up ridiculously long names for all functions if we wanted to - there's plenty of conditions we can think of. `installPackage()` could be `installPackageIfPossible()`, `writeModule()` could be `startWritingModuleAsyncronously()` and so on. But clearly, the existing names have not been big problems that are bothering us - so we should keep using short and concise names. I'm not a fan of switching now.

Reply:
Will `reshowRequiredMainConsoles()` please (or rather, at least, *satisfy*) everyone?

Reply:
Yeah